modyn/tests/supervisor/internal/eval/test_eval_handler.py

```diff
@@ -0,0 +1,94 @@
+from typing import Iterable
+
+import pandas as pd
+from modyn.config.schema.pipeline.evaluation.handler import EvalHandlerConfig
+from modyn.config.schema.pipeline.evaluation.strategy.static import StaticEvalStrategyConfig
+from modyn.supervisor.internal.eval.handler import EvalHandler
+from modyn.supervisor.internal.eval.strategies.abstract import AbstractEvalStrategy, EvalInterval
+
+
+def test_get_eval_requests_after_pipeline() -> None:
```

Comment:
Can you please add a test with the same setup (matrix/most_recent) for after_training as well? This test helped much in understanding the output, so it would be good to have. Thank you!

Reply:
ah yes, sure!

Reply:
the `most_recent_model` determination in the `after_training` case still had conceptual issues, thanks for the pointer, the unit test helped to find those :)

Reply:
perfect!